pulp_python/app/migrations/0017_pythonpackagecontent_size.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 4.2.26 on 2025-11-11 21:43
+
+from django.db import migrations, models, transaction
+
+
+def add_size_to_current_models(apps, schema_editor):
+    """Adds the size to current PythonPackageContent models."""
+    PythonPackageContent = apps.get_model("python", "PythonPackageContent")
+    RemoteArtifact = apps.get_model("core", "RemoteArtifact")
+    package_bulk = []
+    for python_package in PythonPackageContent.objects.only("pk", "size").iterator():
+        content_artifact = python_package.contentartifact_set.first()
+        if content_artifact.artifact:
+            artifact = content_artifact.artifact
+        else:
+            artifact = RemoteArtifact.objects.filter(content_artifact=content_artifact).first()
```

Comment:
Can you do sth like `python_package.contentartifact_set.get().remoteartifact_set`?